### ISSUE TITLE
PDF API GET endpoint

### ISSUE THREAD

**Issue by Digitpark** [crater-invoice/crater#804]
First things first, great work! Lovely to see this project maturing!

For my use case I would like to download a generated PDF through the API, or through the returned PDF URL (in for example invoices) by applying an API authorization header. Currently trying to download the PDF redirects me to the login page, even though applying a (validated) authorization header.

I've found two alternatives, each of which I find insufficient.
1. Set PDF's to be stored on disk and access the PDF through other methods (e.g. SSH)
2. Programmatically recreate a login session and retrieve the PDF through that session.
The first is possible and not too much hassle, but the second seems like beating the purpose of the API.

I'm not familiar with Laravel, so it would be a lot of hassle for me to figure out where to enable allowing the authorization header as an authenticated request to download a PDF url, but maybe it is not much work to enable this.

Another possbile feature would require more labor is a GET endpoint (as the title suggests) for the pdfs. For example `GET /invoices/<id>/pdf`.

tldr; I would like to see the Authorization header for PDF URLs as enough authentication to access a PDF URL, or either a PDF download GET endpoint.

**Comment by mohitpanjwani** [comment]
Thanks for reporting the issue. 

We’re working on creating a new minor release which includes the fix for this one.

**Comment by mohitpanjwani** [comment]
Fixed on https://github.com/crater-invoice/crater/releases/tag/6.0.5

**Comment by Digitpark** [comment]
Thank you for keeping me posted, and thanks for the quick fix!
